### PR TITLE
Sync from DIAL catalog 090820

### PR DESCRIPTION
**Pull request by conradsp**

### CODE REVIEW

File: products/commcare.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "CommCare",
   "description": "CommCare is an easily customizable, open source mobile platform that supports frontline workers in low-resource settings. By replacing paper-based forms, frontline workers can use CommCare to track and support their clients with registration forms, checklists, SMS reminders, and multimedia.",
-  "website": "https://dimagi.com/commcare",
+  "website": "https://www.dimagi.com",
```
Comment (lacabra) [inline products/commcare.json:4]: @conradsp curious to hear your take on this change, since the original address was pointing to the specific page for this product, whereas the new one points to the landing page for the organization (which is also included in the org field below). In my opinion, the website for the product should be specific to the product as much as possible, so I would revert this change

File: products/rasa.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Rasa",
+  "description": "",
```
Comment (lacabra) [inline products/rasa.json:3]: The description field cannot be empty

File: products/community-health-toolkit.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Community Health Toolkit",
```
Comment (lacabra) [inline products/community-health-toolkit.json:2]: This is a duplicate of `products/medic-mobile.json`. I have no strong opinion which one we keep, but not both

**Review by lacabra — CHANGES_REQUESTED** [review]

File: products/commcare.json (hunk 1 above)
Comment (conradsp) [reply products/commcare.json:4]: Agreed - the original is a more specific URL. Updated the data on our end so that it will not be overwritten.

File: products/rasa.json (hunk 2 above)
Comment (conradsp) [reply products/rasa.json:3]: Updating to add description

File: products/community-health-toolkit.json (hunk 3 above)
Comment (conradsp) [reply products/community-health-toolkit.json:2]: Good catch. I'll remove Community Health Toolkit as we have the entry for Medic Mobile already.

File: products/commcare.json (hunk 1 above)
Comment (lacabra) [reply products/commcare.json]: Thanks

File: products/community-health-toolkit.json (hunk 3 above)
Comment (lacabra) [reply products/community-health-toolkit.json]: Looks good now, thanks

**Review by lacabra — APPROVED** [review]
